src/translations/product-list.translations.js

```diff
@@ -0,0 +1,48 @@
+export const SHOW_FILTER_BUTTON_TEXT = [
+  {
```

Comment:
Please, implement transactions objects in the same way as it is done for other entities, look at `homepage.translations.js` as an example

Reply:
fixed